lib/modules/apostrophe-schemas/public/js/user.js

```diff
@@ -1039,10 +1047,10 @@ apos.define('apostrophe-schemas', {
         if (field.required && (!(data[name] && data[name].length))) {
           return setImmediate(_.partial(callback, 'required'));
         }
-        if ((field.max !== undefined) && (data[name] > field.max)) {
+        if (data[name] && field.max !== undefined && data[name] > field.max) {
           return setImmediate(_.partial(callback, 'max'));
         }
-        if ((field.min !== undefined) && (data[name] < field.min)) {
+        if (data[name] && field.min !== undefined && data[name] < field.min) {
```

Comment:
Same issue with 0

Reply:
0 is not an issue here. The value is a string and is checked as a string so the length of `0` is 1 for the required validation at least. What is a concern, though, is that the the comparison of min and max is done as string (input) vs. number (min/max configuration) and so I will add a parseFloat around the value, but inside the check if, because I don't want any more side effects that I cannot foresee by modifying the value on the outside.

Let me know what you think about this. Code coming in shortly.